dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -67,7 +67,7 @@ def create
         restrict_section: params[:restrict_section].nil? ? false : params[:restrict_section]
       }
     )
-    render head :bad_request unless section
+    return head :bad_request unless section.persisted?
```

Comment:
ahh I like that this is the approach instead of switching the 'create' over to the one with '!'.